src/main/java/bio/terra/service/dataset/DatasetTableDao.java

```diff
@@ -36,23 +37,48 @@ public class DatasetTableDao {
   private static final Logger logger = LoggerFactory.getLogger(DatasetTableDao.class);
 
   private static final String sqlInsertTable =
-      "INSERT INTO dataset_table "
-          + "(name, raw_table_name, soft_delete_table_name, row_metadata_table_name, dataset_id, primary_key, bigquery_partition_config) "
-          + "VALUES (:name, :raw_table_name, :soft_delete_table_name, :row_metadata_table_name, :dataset_id, :primary_key, "
-          + "cast(:bigquery_partition_config AS jsonb))";
+      """
+  INSERT INTO dataset_table
+  (name, raw_table_name, soft_delete_table_name, row_metadata_table_name, dataset_id, primary_key, bigquery_partition_config)
+  VALUES (:name, :raw_table_name, :soft_delete_table_name, :row_metadata_table_name, :dataset_id, :primary_key,
+  cast(:bigquery_partition_config AS jsonb))
+  """;
+
   private static final String sqlInsertColumn =
-      "INSERT INTO dataset_column "
-          + "(table_id, name, type, array_of, required, ordinal) "
-          + "VALUES (:table_id, :name, :type, :array_of, :required, :ordinal)";
+      """
+  INSERT INTO dataset_column
+  (table_id, name, type, array_of, required, ordinal)
+  VALUES (:table_id, :name, :type, :array_of, :required, :ordinal)
+  """;
+
   private static final String sqlSelectTable =
-      "SELECT id, name, raw_table_name, soft_delete_table_name, row_metadata_table_name, primary_key, bigquery_partition_config::text, "
-          + "(bigquery_partition_config->>'version')::bigint AS bigquery_partition_config_version "
-          + "FROM dataset_table WHERE dataset_id = :dataset_id";
+      """
+  SELECT t.id table_id,
+         t.name table_name,
+         t.raw_table_name table_raw_table_name,
```

Comment:
I guess it's nice to be consistent but only aliasing duplicate names would've also worked here and would be less verbose. It's a SQL error to have duplicate column names in a result (I believe) so you wouldn't have a case where names overlap silently.

Reply:
Yeah, I explicitly wanted to be consistent here